docs/source/tools/acquire.rst

```diff
@@ -63,6 +72,44 @@ These arguments do the following:
 
 You can specify these arguments multiple times for every file, directory or glob you want to collect.
 
+Volatile acquisition
+--------------------
+
+*new in Acquire 3.11*
+
+Use ``--volatile-profile`` to obtain artefacts that are not persistent on disk but are located in RAM.
+Volatile Windows artifacts are stored in de ``$metadata$`` folder in the resulting archive.
+Windows volatile artifacts are acquired through the use of internal Windows commands and the Python ``ctypes`` interface.
+For Linux systems, ``proc`` and ``sys`` are acquired and stored under ``/proc/1/...`` or ``/sys/fs/...`` respectively.
+
+Volatile Profiles
+^^^^^^^^^^^^^^^^^
+
+Like regular *profiles*, the *volatile profiles* allow you to run predefined groups of modules.
+The following volatile profiles are available:
+
+* ``default``: Collect a set of volatile information.
+* ``extensive``: Collect a lot, if not all, the volatile information. Usually the modules that either take a lot of time or are very involved.
+* ``none``: Do not collect any volatile information.
```

Comment:
This is the default behaviour without specifying `--volatile-profile` right? It might be a good idea to convey that here as well.